cfme/tests/containers/test_provider_openscap_policy_attached.py

```diff
@@ -0,0 +1,87 @@
+import dateparser
+import random
+import pytest
+
+from cfme.configure.tasks import delete_all_tasks
+from cfme.containers.provider import (ContainersProvider, refresh_and_navigate)
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.wait import wait_for
+from collections import namedtuple
+
+
+pytestmark = [
+    pytest.mark.meta(server_roles='+smartproxy'),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1),
+    pytest.mark.provider([ContainersProvider], scope='function')]
+
+AttributeToVerify = namedtuple('AttributeToVerify', ['table', 'attr', 'verifier'])
+
+TESTED_ATTRIBUTES__openscap = (
+    AttributeToVerify('configuration', 'OpenSCAP Results', bool),
+    AttributeToVerify('configuration', 'OpenSCAP HTML', lambda val: val == 'Available'),
+    AttributeToVerify('configuration', 'Last scan', dateparser.parse),
+    AttributeToVerify('compliance', 'Status', lambda val: val.lower() != 'never verified'),
+    AttributeToVerify('compliance', 'History', lambda val: val == 'Available')
+)
+
+
+@pytest.fixture(scope='function')
+def delete_all_container_tasks():
+    delete_all_tasks('AllTasks')
+
+
+@pytest.fixture(scope='function')
+def random_image_instance(appliance):
+    collection = appliance.collections.container_images
+    return random.sample(collection.all(), 1).pop()
+
+
+def get_table_attr(instance, table_name, attr):
+    # Trying to read the table <table_name> attribute <attr>
+    view = refresh_and_navigate(instance, 'Details')
+    table = getattr(view.entities, table_name, None)
+    if table:
+        return table.read().get(attr)
+
+
+@pytest.mark.polarion('10068')
+def test_check_compliance_provider_policy(provider, soft_assert,
+                                        random_image_instance):
+
+    # unassign OpenSCAP policy from chosen Image
+    random_image_instance.unassign_policy_profiles('OpenSCAP profile')
+
+    # assign policy from provider view
+    provider.assign_policy_profiles('OpenSCAP profile')
+
+    # Delete all old tasks in order to get fresh start
+    delete_all_container_tasks()
+
+    # Prform SSA Scan then check compliance with last know configuration
+    random_image_instance.perform_smartstate_analysis(wait_for_finish=True)
```

Comment:
**REQUIRED**
All this stuff above should be in a fixture.

Reply:
add All of above to new Fixture openscap_assigned_rand_image